src/moleculeSimulator/Simulator.cpp

```diff
@@ -0,0 +1,61 @@
+//
+// Created by daniel on 24.04.24.
+//
+
+#include "Simulator.h"
+
+void Simulator::calculateF() {
+    for (auto &p_i : particles) {
+        p_i.setOldF(p_i.getF());
+        p_i.setF({0,0,0});
+        for (auto &p_j : particles) {
+            if(&p_i != &p_j) {
+                p_i.setF(p_i.getF() + force.compute(p_i, p_j));
+            }
+        }
+    }
```

Comment:
This is simple and clear, but does not yet use Newton's third law.
You should try to use it for further worksheets when we increase the number of particles.